src/werkzeug/http.py

```diff
@@ -1099,19 +1175,19 @@ def _parse_pairs():
 
 
 def dump_cookie(
-    key,
-    value="",
-    max_age=None,
-    expires=None,
-    path="/",
-    domain=None,
-    secure=False,
-    httponly=False,
-    charset="utf-8",
-    sync_expires=True,
-    max_size=4093,
-    samesite=None,
-):
+    key: str,
+    value: Union[str, bytes] = "",
+    max_age: Optional[int] = None,
+    expires: Optional[int] = None,
+    path: str = "/",
+    domain: Optional[str] = None,
+    secure: Optional[bool] = False,
```

Comment:
Not sure if this should be nullable.

Reply:
Whilst `None` would work (same as `False`), I think `bool` is the better type hint as it is clearer that cookies can be secure or not.